src/styles/landing-page.less

```diff
@@ -36,6 +37,55 @@
     flex: 1 1 0%; // for .word-break()
     .word-break();
   }
+  .dropdown-menu {
+    margin-top: 0;
+    padding: 0;
+    width: 100%;
+    > li > a.catalog-search-match {
+      border-color: @color-pf-black-300;
+      border-width: 0 0 1px 0;
+      display: flex;
+      line-height: 1.5;
+      padding: 3px 10px 3px 5px;
+      white-space: normal;
+    }
+    > li > a.catalog-search-show-all {
+      background-color: @color-pf-black-200;
+      color: @link-color;
+      padding: 5px 0;
+      text-align: center;
+    }
+
+    // important needed to override styles declared important in PatternFly
+    .active > a.catalog-search-match, :focus{
```

Comment:
I'm confused by the :focus that's not attached to an element.  Are you sure you want to override :focus styles for any descendent of .dropdown-menu?

Nit: need a space between s and {

Reply:
This code was moved from below such that it only applies to item under  catalog-search not necessarily on the landing page. I questioned it as well but decided to leave it as it was. I'll investigate further.

Reply:
Turns out the :focus is not necessary at all. It doesn't change a thing.